core/deployment/src/main/resources/dev-ui/qwc-qosdk-controllers.js

```diff
@@ -0,0 +1,65 @@
+import { LitElement, html, css, nothing} from 'lit';
+import { JsonRpc } from 'jsonrpc';
+import '@vaadin/grid';
+import '@vaadin/details';
+import '@vaadin/list-box';
+import '@vaadin/item';
+import {columnBodyRenderer} from '@vaadin/grid/lit.js';
+export class ControllersWebComponent extends LitElement {
```

Comment:
For consistency, and to make sure we do not "clash" classNames, this should be named the camel-case of the file, so `QwcQoSDKControllers`.

Reply:
Done.